src/allowlist/src/main.rs

```diff
@@ -76,6 +84,8 @@ enum InitError {
     Connection(RunError<RedisError>),
     #[error("Could not start a Redis connection manager: {0}")]
     ConnectionManager(RedisError),
+    #[error("Failed to install Ctrl+C handler: {0}")]
+    CtrlCHandler(std::io::Error),
```

Comment:
`Ctrl+C` is just sending `SIGINT`. To be consistent with the naming of `SIGTERMHandler` below, I'd rename this `SIGINTHandler`.

You could also go for `SigtermHandler` and `SigintHandler` as this follows the Rust naming convention more precisely, but I understand why one would think not following it is more appropriate here.

Reply:
> `Ctrl+C` is just sending `SIGINT`

If running this container in Docker Compose per the README, `Ctrl+C` will actually send Docker sends `SIGTERM` (https://docs.docker.com/compose/faq/#why-do-my-services-take-10-seconds-to-recreate-or-stop)

> `SigtermHandler` ... follows the Rust naming convention

Addressed in ece33d7